development/build/manifest.js

```diff
@@ -18,6 +18,23 @@ const { getEnvironment, getBuildName } = require('./utils');
 
 module.exports = createManifestTasks;
 
+async function loadManifestFlags() {
+  try {
+    return JSON.parse(
+      await fs.readFile(
+        path.join(__dirname, '../../.manifest-flags.json'),
+        'utf8',
+      ),
+    );
+  } catch (error) {
+    return { remoteFeatureFlags: {} };
+  }
+}
+
+const manifestFlags = loadManifestFlags();
```

Comment:
Did you mean to make `loadManifestFlags` synchronous?

Reply:
Does it make sense to use `fs.readFileSync` and make it synchronous anyway?

Reply:
@DDDDDanica I don't think we resolved this yet